### ISSUE TITLE
Missing extensions

### ISSUE THREAD

**Issue by zoggy** [mirage/ocaml-magic-mime#3]
The `mime.types` file is rather limited, it does not even include `.ml` and `.mli` files :)
Would you accept patches for additional extensions ? These could be stored in another file not to mess with the "official" file.

**Comment by talex5** [comment]
Perhaps we should use the freedesktop.org shared MIME database as the source: http://www.freedesktop.org/wiki/Software/shared-mime-info/

This is what e.g. file managers on Linux systems use. It has:

```
<mime-type type="text/x-ocaml">
  <_comment>OCaml source code</_comment>
  <glob pattern="*.ml"/>
  <glob pattern="*.mli"/>
</mime-type>
```

**Comment by avsm** [comment]
I picked this one as it was reasonably small, since it is crunched into the service.  Pulling in the full database could also be done in a similar style to `ocaml-uri`, which has separate ocamlfind packages for a large and a minimal one.  In the meanwhile, feel free to submit additions to the `mime.types` file that's in here right now!

**Comment by avsm** [comment]
fixed in #4
